opm/input/eclipse/Schedule/Well/Well.hpp

```diff
@@ -536,6 +536,7 @@ class Well {
     const std::string& groupName() const;
     Phase getPreferredPhase() const;
 
+    std::size_t getNoConnections() const;
```

Comment:
Would you mind naming this `getNumConnections()` instead, please?  We might also/alternatively consider having a predicate of the form
```C++
bool haveConnections() const;
```
with an implementation along the lines of
```C++
bool haveConnections() const { return !this->connections.empty(); }
```
if we're **mostly** concerned about the case `getNoConnections() == 0`.

Reply:
i went for bool hasConnections() - other methods in there use 'has' instead of 'have' already.